game.py

```diff
@@ -70,9 +73,168 @@ def self_play_game(model,
     action_distributions = np.array(action_distributions)
     return states, v, action_distributions
 
+def play_many_vs_random_games(num_games,
```

Comment:
We should just have one function that takes in two models/strategies (one that goes first and the other that goes second), instead of having `vs_random` and `smart_vs_smart` etc. It should be pretty simple to have a random model, and we can just pass that in if we ever want to do games versus a random strat.